src/test/isolation2/expected/sync_guc.out

```diff
@@ -153,3 +153,38 @@ RESET
 (4 rows)
 4q: ... <quitting>
 
+-- TEST 4: make sure GUC sync failure doesn't leave broken temp tables
+1: create temp table sync_temp_table(a int) distributed by (a);
+CREATE
+-- Cause panic on segment from another session
+2: select gp_inject_fault('create_function_fail', 'panic', dbid) from gp_segment_configuration where content=0 and role='p';
+ gp_inject_fault 
+-----------------
+ Success:        
+(1 row)
+2: create function my_function() returns void as $$ begin end; $$ language plpgsql;
+ERROR:  fault triggered, fault name:'create_function_fail' fault type:'panic'  (seg0 192.168.0.101:6002 pid=347506)
+2q: ... <quitting>
+-- Attempting to change GUC
+!\retcode gpconfig -c log_min_messages -v 'warning' -m 'notice';
+(exited with code 0)
+!\retcode gpstop -u;
+(exited with code 0)
+
+-- Query to execute GUC sync (GUC sync fails, query succeeds)
+1: select 1;
+ ?column? 
+----------
+ 1        
+(1 row)
+-- Should fail with "relation does not exist"
```

Comment:
Shouldn't there be a message about this?
```
WARNING:  Any temporary tables for this session have been dropped because the gang was disconnected (session id = ...)  (seg... 127.0.1.1:... pid=...)
```

Reply:
It seems like isolation tests filter warnings somehow (there are no warning in other isolation tests as well). If this exact commands are run manually, the warning is there.